bedrock/firefox/views.py

```diff
@@ -541,6 +542,17 @@ def get_template_names(self):
         return [template]
 
 
+def fxos(request):
```

Comment:
This looks ok to me but would be good to get either @pmclanahan or @jgmize to give their approval. Also, do we need to do some waffle magic here still?